### ISSUE TITLE
use of ip address

### ISSUE THREAD

**Issue by gnamienprince** [franzose/laravel-smpp#18]
I would like to know if this code uses ip addresses and ports to send sms without going through a provider?

**Comment by franzose** [comment]
@gnamienprince yes, you can see this in the configuration: https://github.com/franzose/laravel-smpp/blob/master/config/laravel-smpp.php. It is a direct connection to an SMS provider via the SMPP protocol without any aggregators.
